### ISSUE TITLE
 full path of the remote rp sandbox

### ISSUE THREAD

**Issue by euhruska** [radical-collaboration/extasy-grlsd#39]
is it possible to print on the local output the full path of the remote rp sandbox?

**Comment by vivek-bala** [comment]
No, not at the moment. Can you tell me how you want to use that path?

**Comment by euhruska** [comment]
usability, telling the scientist that the raw data is there

**Comment by vivek-bala** [comment]
Hmmm. I think the default has been that if the user wants to copy or download it, EnTK will take care of the bookkeeping. But I suppose the user could also just want to know where the data is. Maybe to download them post-mortem?

I'll think about what the appropriate API could be and report back. One candidate is https://github.com/radical-cybertools/radical.entk/issues/190.

**Comment by euhruska** [comment]
to debug I have to look every time into the units, so that information is useful for me. When running many extasy runs, then I lose track which run was which rp.session.
